woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2980,7 +2966,7 @@ def test_infer_missing_logical_types_force_infer(sample_df):
     parsed_logical_types = _infer_missing_logical_types(
         sample_df, force_logical_types, existing_logical_types
     )
-    assert parsed_logical_types["age"] == Double()
+    assert parsed_logical_types["age"] == IntegerNullable()
```

Comment:
Should we be asserting with `isinstance` here instead of checking object equality?